react/ButtonGroup/ButtonGroup.js

```diff
@@ -0,0 +1,17 @@
+import styles from './ButtonGroup.less';
+import React, { PropTypes } from 'react';
+
+export default function ButtonGroup({ children }) {
+  return (
+    <div>{children.map(child => React.cloneElement(child, { className: styles.button }))}</div>
```

Comment:
I did initially wrap it but adding an extra element to achieve some extra margin felt a bit wrong.  I am happy to change it back if that's everyone's preferred approach?